doc/lsst.source.injection/reference/41_match_injected_outputs.rst

```diff
@@ -0,0 +1,201 @@
+.. _lsst.source.injection-ref-inspect:
+
+========================
+ Match Injected Outputs
+========================
+
+-----------------------
+ Process injected data
+-----------------------
```

Comment:
Maybe something like "Consolidate and match source injection catalogs"?

Reply:
I think you need to add an introductory paragraph below the subtitle, stating what this page will cover. Everything else then should go below in their own headed sections. I'd recommend staying away from words like "process", as it's so generic it can mean multiple things. For your purposes, "reduce data through step 3", "consolidate per-patch catalogs into tract-level catalogs" and "match catalogs" should be your go-to phrases.